VocaDb.Migrations/Migrations.cs

```diff
@@ -5,6 +5,15 @@ namespace VocaDb.Migrations {
 
 	// Migration version format: YYYY_MM_DD_HHmm
 
+	[Migration(2020_03_22_0000)]
+	public class VenuesRegionCode : AutoReversingMigration {
+
+		public override void Up() {
+			Create.Column("RegionCode").OnTable(TableNames.Venues).AsString(10).NotNullable().WithDefaultValue(string.Empty);

```

Comment:
Could you rename this to AddressCountryCode ? I think it would be more descriptive. 

By the way, when making changes to migrations in feature branches like this, it's ok to update the migration. No need to create a new migration, because it hasn't been applied anywhere yet.